app/grandchallenge/pages/views.py

```diff
@@ -86,6 +88,48 @@ def test_func(self):
         return page.can_be_viewed_by(user=user)
 
 
+class PagePandoc(
+    UserPassesTestMixin, ChallengeFilteredQuerysetMixin, DetailView
+):
+    model = Page
+    permission_required = "view_page"
+    raise_exception = True
+    login_url = reverse_lazy("account_login")
+
+    raw = None
+
+    def test_func(self):
+        return self.request.user.is_staff
+
+    def get_object(self, *args, **kwargs):
+        if not self.kwargs["format"].startswith("markdown"):
+            raise PermissionDenied()
+        return super().get_object(*args, **kwargs)
+
+    def get_template_names(self):
+        if self.raw:
+            return ["pages/page_pandoc_detail.html"]
+        else:
+            return super().get_template_names()
```

Comment:
Why do we need a separate raw view? It shows the same converted content, just in isolation, right?

Reply:
Yes, this is what the user will be editing, so using different Markdown formats will result in different sources, some of which will be easier to work with than others. A big one is how tables are represented.

Reply:
Sorry, I don't really follow. The user is editing their page in the update view. They won't even see this converted version for now, will they?

Reply:
No, but they will once we migrate to using markdown, I want to compare what it will look like to them without having to guess, then migrate, then find out I made a bad choice afterwards.